week02/이성진/todo/src/component/TodoItem.jsx

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+
+function TodoItem(props){
```

Comment:
```suggestion
function TodoItem({todoItem}){
```

여기도 동일하게 구조분해할당을 사용하는 것이 좋아보여요